src/test/java/gov/usgs/wma/waterdata/TransformDaoIT.java

```diff
@@ -0,0 +1,130 @@
+package gov.usgs.wma.waterdata;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase.Replace;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
+import org.springframework.test.context.support.DirtiesContextTestExecutionListener;
+import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.github.springtestdbunit.TransactionDbUnitTestExecutionListener;
+import com.github.springtestdbunit.annotation.DatabaseSetup;
+import com.github.springtestdbunit.annotation.DbUnitConfiguration;
+
+@SpringBootTest(webEnvironment=WebEnvironment.NONE,
+		classes={
+			DBTestConfig.class,
+			TransformDao.class})
+@DatabaseSetup("classpath:/testData/transformDb/groundwaterStatisticalDailyValue/")
+@ActiveProfiles("it")
+@TestExecutionListeners({ DependencyInjectionTestExecutionListener.class,
+		DirtiesContextTestExecutionListener.class,
+		TransactionalTestExecutionListener.class,
+		TransactionDbUnitTestExecutionListener.class })
+@DbUnitConfiguration(
+		dataSetLoader=FileSensingDataSetLoader.class,
+		databaseConnection={"transform"}
+)
+@AutoConfigureTestDatabase(replace=Replace.NONE)
+@Transactional(propagation=Propagation.NOT_SUPPORTED)
+@Import({DBTestConfig.class})
+@DirtiesContext
+public class TransformDaoIT {
```

Comment:
The duplicated annotations, properties, and setup could be extracted to an abstract base dao integration class and shared between the two.

Reply:
Done, see the new abstract testing class `BaseTestDao`